tests/spread/general/stage_snaps/task.yaml

```diff
@@ -1,10 +1,8 @@
 summary: stage-snaps
 
-# We currently only have core18 on a stable channel
 systems:
-  - ubuntu-18.04-64
-  - ubuntu-18.04-amd64
-  - ubuntu-18.04-i386
+  - ubuntu-20.04*
+  - ubuntu-18.04*
```

Comment:
Is this sufficient?  Below it is setting up a project with `snapcraft init`, which would presumably create a core18 basedsnap on all systems.  You'd need a core20 based snap to exercise v2 plugins.

Reply:
Indeed not. A normal pattern for us is to use `set_base` to correctly set the base to match the host. Added, thanks :)